Code/Framework/AzCore/AzCore/std/allocator_traits.h

```diff
@@ -31,7 +31,7 @@ namespace AZStd
 
         //! pointer
         //! Attempts to use the C++ standard allocator_type::pointer typedef if avialable,
-        //! otherwise attempt to fallback to the AZStd pointer_type typedef.
+        //! otherwise attempt to fallback to the AZStd pointer typedef.
```

Comment:
Can you do me a favor and fix the typo in the term avialable?

Reply:
Fixed.